Demo/src/components/ImageCellHeader/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {Image, Text, View} from 'react-native';
+import ImageCellHeaderStyles from './styles';
+import {Queue} from 'react-native-spacing-system';
+
+export type ImageCellHeaderProps = {
+  profileUrl?: string;
+  authorName?: string;
+};
+
+const ImageCellHeader: React.FC<ImageCellHeaderProps> = (
+  props: ImageCellHeaderProps,
+) => {
+  return (
+    <View style={ImageCellHeaderStyles.headerContainerStyle}>
+      <Image
+        style={ImageCellHeaderStyles.profileImageStyle}
+        source={
+          props.authorName
+            ? {uri: props.profileUrl}
+            : require('../../assets/images/profileImagePlaceholder.jpg')
```

Comment:
require('../../assets/images/profileImagePlaceholder.jpg') should be in the Images file. 
You have already done it in the BaseUI section.